### PR TITLE
[TASK]: Code changed

### PR DESCRIPTION
**Pull request by nayanasQA**

### CODE REVIEW

File: classes/task/insert_recordings.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function execute() {
             } catch (\moodle_exception $error) {
                 mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
             }
-             if (!empty($recordings) && !empty($recordings->recording_files{0})) {
+            if (!empty($recordings) && !empty($recordings->recording_files{0})) {
```
Comment (Naveen1895) [inline classes/task/insert_recordings.php]: Can we have this entire block inside try-catch block ???

Comment (nayanasQA) [reply classes/task/insert_recordings.php]: @Naveen1895 code changed and updated.

File: classes/task/insert_recordings.php [hunk 2]
```diff
@@ -65,10 +62,15 @@ public function execute() {
...
-                if(!empty($insert_record)) {
+                $zoom_recordings = $DB->insert_record('zoom_recordings', $record);
+                if(is_int($zoom_recordings)) {
                     $DB->update_record('event', (object)['id' => $value->id, 'recording_created' => 1]);
                 }
+            } else {
+                mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
+            }
+            } catch (\moodle_exception $error) {
+                mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
```
Comment (Naveen1895) [inline classes/task/insert_recordings.php]: @nayanalinkstreet should we not print the caught exception here?

Comment (nayanasQA) [reply classes/task/insert_recordings.php]: @Naveen1895 updated

File: classes/task/insert_recordings.php [hunk 3]
```diff
@@ -51,10 +51,7 @@ public function execute() {
             try {
                 $this->disable_download_in_stream($value->meeting_id);
                 $recordings = $service->get_meeting_recording($value->meeting_id);
-            } catch (\moodle_exception $error) {
-                mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
-            }
-             if (!empty($recordings) && !empty($recordings->recording_files{0})) {
+                 if (!empty($recordings) && !empty($recordings->recording_files{0})) {
```
Comment (Naveen1895) [inline classes/task/insert_recordings.php]: @nayanalinkstreet Can you fix the indentation issue ?

Comment (nayanasQA) [reply classes/task/insert_recordings.php]: @Naveen1895 Indentation has done

**Comment by Naveen1895** [comment]
What about the version ?

**Review by nayanasQA — COMMENTED** [review]
> What about the version ?

@Naveen1895 updated the version.

**Review by meyyappanv — APPROVED** [review]

**Review by Naveen1895 — APPROVED** [review]

**Pull request merged**